rackcity/utils/change_planner_utils.py

```diff
@@ -289,13 +297,65 @@ def get_page_count_response_for_cp(request, change_plan):
     return JsonResponse({"page_count": page_count})
 
 
-def get_changes_on_asset(asset, assetCP):
+def network_connections_have_changed(asset, asset_cp):
+    network_ports_cp = NetworkPortCP.objects.filter(asset=asset_cp)
+    network_ports = NetworkPort.objects.filter(asset=asset)
+    for network_port_cp in network_ports_cp:
+        # Get Asset corresponding to this CP connection
+        connected_port_cp = network_port_cp.connected_port
+        if connected_port_cp:
+            destination_host_in_cp = connected_port_cp.asset.related_asset
+        else:
+            destination_host_in_cp = None
+        # Get Asset corresponding to this live connection
+        connected_port_live = network_ports.get(
+            port_name=network_port_cp.port_name
+        )
+        if connected_port_live:
+            destination_host_live = connected_port_live.asset
+        else:
+            destination_host_live = None
+        # Check for match
+        if destination_host_in_cp != destination_host_live:
```

Comment:
my brain is tired. would this miss changes in destination port names? aka connected to same hostname but a different port?

Reply:
good catch yeah I'll update